scripts/table.js

```diff
@@ -2,6 +2,9 @@
 
 const form = document.getElementById("form");
 
+const url = "https://0c43-2401-ff80-1880-7e2a-fd19-2c2a-fad1-6019.ngrok.io";
```

Comment:
take it from url query parameter if present.


If not present, the earlier behaviour (no communication with server, just showing in table) should work.